CHANGELOG.md

```diff
@@ -4,6 +4,352 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.23.0-rc0](https://github.com/go-gitea/gitea/releases/tag/v1.23.0-rc0) - 2024-12-16
+
+* BREAKING
+  * Use camo.Always instead of camo.Allways (#32097)
+  * Remove SHA1 for support for ssh rsa signing (#31857)
+  * Use UTC as default timezone when schedule Actions cron tasks (#31742)
+  * Clear up old Actions logs (#31735)
+  * Fix OIDC introspection authentication (#31632)
+  * Rename project `board` -> `column` to make the UI less confusing (#30170)
+
+* SECURITY
+  * Include file extension checks in attachment API (#32151)
+  * Update JS and PY dependencies (#31940)
```

Comment:
It could be beside other "update dependency"

Reply:
I'm OK to keep them here or move to the refactor section because it mentioned there are some securities on these old versions.

Reply:
IIRC, `Update JS and PY dependencies (#31940)` updates the development dependency, it doesn't affect end users.

So:

* merge the dependency update with others.
* mention that "Includes all security fixes backported to 1.22"

Reply:
e18b97da7a4ecc82b44955fd968453307213fe6d